src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -596,7 +596,7 @@ private function buildEmailMessage(
                 Query::limit(\count($bccTargets)),
             ]);
             foreach ($bccTargets as $bccTarget) {
-                $bcc[] = ['email' => $bccTarget['identifier']];
+                $bcc[] = ['email' => $bccTarget['identifier'], 'name' => $bccTarget['name'] ?? $bccTarget['identifier']];
```

Comment:
I'm worried this is going to send to the provider the user's email address as the name. So the raw email message will have something like:

```
To: "Joe" <joe@example.com>
CC: "jane@example.com" <jane@example.com>
```

It might be best to just pass an empty string.